.github/workflows/ms.network.azurefirewalls.yml

```diff
@@ -88,6 +88,7 @@ jobs:
       - job_module_pester_validation
     strategy:
       fail-fast: false
+      max-parallel: 1
```

Comment:
Bis dislike :D Unless there is a limit on the number of Firewalls we can have at any given time, we should try to find another solution. 
In this case the best would be to introduce a new subnet so that both parameter files use dedicated instances. It's something we do e.g. also for Public IPs (see LoadBalancer example).
If you need help with that, let me know - we just did something similar for a different workload (API management) today

Reply:
Also - the same would happen in case of the ADO pipelines